docs/rules/id-blacklist.md

```diff
@@ -1,41 +1,29 @@
-# Blacklist certain identifiers to prevent them being used (id-blacklist)
+# Disallow Specified Identifiers (id-blacklist)
 
 > "There are only two hard things in Computer Science: cache invalidation and naming things." — Phil Karlton
 
-Bad names can lead to hard to decipher code. Using generic names, such as `data` don't infer much about the code and the values it receives. This rule allows you to configure a blacklist of bad identifier names, that you don't want to see in your code.
+Bad names can lead to hard-to-decipher code. Generic names, such as `data`, don't infer much about the code and the values it receives. This rule allows you to configure a blacklist of bad identifier names, that you don't want to see in your code.
 
 ## Rule Details
 
-This rule compares assignments and function definitions to a provided list of identifier names. If the identifier is present in the list, it will return an error.
-
-This rule will catch blacklisted identifiers that are:
-
-- variable declarations
-- function declarations
-- object properties
-
-It will not catch blacklisted identifiers that are:
-
-- function calls (so you can still use functions you do not have control over)
-- object properties (so you can still use objects you do not have control over)
-
+This rule disallows specified identifiers in assignments and `function` definitions.
```

Comment:
@keithamus What do you think of something like the following to keep the contrasting bulleted lists but decrease the length of sentences?

This rule disallows specified identifiers in:
* variable declarations
* function declarations
* assignment statements [however to make it clear the lhs including object properties]

This rule does not apply to identifiers which you might not have control over:
* function calls
* object properties [however to make it clear other than assignment mentioned above]